shell/scripts/storage_stats.rs

```diff
@@ -0,0 +1,468 @@
+        let block_latency = m.get_block_latency(0).unwrap();
+        self.block_latencies_total += block_latency as usize;
+
+        let stats: String = format!(
+            "{} {} {} {} {} {} {} {} {} {} {} {} {} {} {} {} {} {} ",
+            block_nr,
+            usage,
+            block_latency,
+            self.block_latencies_total,
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::Set,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::Get,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::GetByPrefix,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::GetKeyValuesByPrefix,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::GetContextTreeByPrefix,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::GetHistory,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::Mem,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::DirMem,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::Copy,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::Delete,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::DeleteRecursively,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::Commit,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::Checkout,
+                &report.perf_stats.global
+            ),
+            self.generate_stats_for_merkle_action(
+                MerkleStorageAction::BlockApplied,
+                &report.perf_stats.global
+            )
+        );
+
+        writeln!(&mut self.output, "{}", stats).unwrap();
+    }
+}
+
+fn main() -> Result<(), Error> {
+    let params = Args::read_args();
+
+    let out_dir = PathBuf::from(params.output.as_str());
+    let key_value_db_path = out_dir.join("key_value_store");
+
+    let actions_storage_path = PathBuf::from(params.input.as_str());
+
+    let _ = fs::remove_dir_all(&out_dir);
```

Comment:
@mat13mn 
this i suppressed error,
please do at least: `let _ = fs::remove_dir_all(&out_dir)?;`